src/presenter.ts

```diff
@@ -34,12 +37,25 @@ export const presentProfile = (
 
   const { backgroundColor, highlightColor } = getColors(subject, store);
 
+  let pronouns = store.anyJS(subject, ns.solid('preferredSubjectPronoun')) || ''
```

Comment:
I think angelo mentioned making this a function.  The weird thing is I thought I saw the function, but I can't seem to find it.